health-services/project-factory/src/server/generateFlowClasses/user-generateClass.ts

```
@@ -0,0 +1,206 @@
+                dynamicColumns: null
+            }
+        };
+
+        logger.info(`SheetMap generated for template type: ${type}`);
+        return sheetMap;
+    }
+
+
+    static getReadMeData(readMeConfig: any, readMeColumnHeader: any, localizationMap: any) {
+        const dataArray = [];
+        for (const text of readMeConfig?.texts) {
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)